### PR TITLE
lively: Add version 2.0.5.0

### PR DESCRIPTION
**Pull request by Zliced13**
A free and open-source software that allows users to set animated desktop wallpapers and screensavers. 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/lively.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "https://github.com/rocksdanister/lively/releases/download/v2.0.5.0/lively_setup_x86_full_v2050.exe",
+    "hash": "00446677990be6841dd83b9e3b736d71a70a094f3babbdfb5fb9139f97d95b94",
+    "innosetup": true,
+    "shortcuts": [
+        [
+            "Lively.exe",
+            "Lively"
+        ]
+    ],
+    "pre_uninstall": "Remove-ItemProperty 'HKCU:\\Software\\Microsoft\\Windows\\CurrentVersion\\Run' -Name 'Lively' -ErrorAction 'SilentlyContinue' -Force",
```
Comment (HUMORCE) [inline bucket/lively.json:19]:
```suggestion
```

File: bucket/lively.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "Microsoft .NET Desktop Runtime": "extras/windowsdesktop-runtime",
+        "VCRedist": "extras/vcredist"
+    },
+    "url": "https://github.com/rocksdanister/lively/releases/download/v2.0.5.0/lively_setup_x86_full_v2050.exe",
+    "hash": "00446677990be6841dd83b9e3b736d71a70a094f3babbdfb5fb9139f97d95b94",
+    "innosetup": true,
+    "shortcuts": [
+        [
+            "Lively.exe",
+            "Lively"
```
Comment (HUMORCE) [inline bucket/lively.json:16]:
```suggestion
            "Lively Wallpaper"
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/lively.json (hunk 1 above)
Comment (Zliced13) [reply bucket/lively.json:19]: Why not remove it? Why keep it there?

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**